docs/tutorials/dicom.ipynb

```diff
@@ -104,7 +104,7 @@
       "source": [
         "## Overview\n",
         "\n",
-        "This tutorial shows how to use [dicom](https://github.com/tensorflow/io/tree/master/tensorflow_io/dicom) in TensorFlow IO to decode DICOM files with TensorFlow."
+        "This tutorial shows how to use [dicom](https://github.com/tensorflow/io/blob/master/tensorflow_io/core/python/ops/dicom_ops.py) in TensorFlow IO to decode DICOM files with TensorFlow."
```

Comment:
Can we point to https://www.tensorflow.org/io/api_docs/python/tfio/dicom
When this is published on the site? @MarkDaoust

Reply:
Thanks @lamberta, the PR has been updated.